backend/worker.yaml

```diff
@@ -3,6 +3,10 @@ api_version: 1
 threadsafe: true
 service: worker
 
+libraries:
```

Comment:
You don't need this property anymore, once you added to the requirements files. But, be aware that you have to execute the pip install before to execute the application.

Reply:
Nice